spec/lib/crawler/data/crawl_result/html_spec.rb

```diff
@@ -168,7 +170,7 @@
 
     it 'should return a set of links from the document' do
       expect(links).to be_kind_of(Set)
-      expect(links.count).to eq(7)
+      expect(links.count).to eq(6)
```

Comment:
This number change is intentional; Nokogiri would extract links even if they were malformed (e.g. with `:href` instead of `href`), while Jsoup doesn't extract these.